src/components/fx/hover.js

```diff
@@ -25,6 +25,8 @@ var Registry = require('../../registry');
 var helpers = require('./helpers');
 var constants = require('./constants');
 
+var legend = require('../legend');
```

Comment:
We usually use `Legend` (not `legend`) when requiring the legend module. 

Personally, I prefer require just the methods we need. I think here only:

```js
var legendSupplyDefaults = require('../legend/defaults');
var legendDraw = require('./legend/draw');
```

would suffice.

Reply:
Done in 9ea8bfdab29495a2eaf96884a36dff71a38aff67

Reply:
Resolving. Done in https://github.com/plotly/plotly.js/pull/4620/commits/9ea8bfdab29495a2eaf96884a36dff71a38aff6.